src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp

```diff
@@ -2520,6 +2520,12 @@ void MacroAssembler::resolve_global_jobject(Register value, Register tmp1, Regis
   bind(done);
 }
 
+void MacroAssembler::breakpoint(uint16_t hint_imm16) {
+  // BRK with hint
+  const uint32_t c = 0xD4200000 | (((uint32_t)hint_imm16) << 5);
```

Comment:
aarch64 already provides `brk` at assembler [1],  for readability, maybe we can just simply use `brk(hint_imm16)` here.

1. https://github.com/openjdk/jdk/blob/969a6b9fd7f7afc60250309f3ada205c1473cf8e/src/hotspot/cpu/aarch64/assembler_aarch64.hpp#L1036